docs/source/markdown/podman-container-clone.1.md

```diff
@@ -88,15 +88,16 @@ On a multi-core system, the shares of CPU time are distributed over all CPU
 cores. Even if a container is limited to less than 100% of CPU time, it can
 use 100% of each individual CPU core.
 
-For example, consider a system with more than three cores.
-container **{C0}** is started with **-c=512** running one process, and another container
-**{C1}** with **-c=1024** running two processes, this can result in the following
+For example, consider a system with more than three cores. If you start one
```

Comment:
no pronouns in man pages please

Instead "If the container **(Co}** is started with..."

Reply:
And I see the "you" bit is older in other man pages,  Please rework those too and apologies for the added headache.

@edsantiago in your marvelous man page parser, could we add a check for the word "you" and flag that?

Reply:
I removed the pronoun. I also removed the strange curly brace around the container name.